.github/workflows/ci.yml

```diff
@@ -1,23 +1,41 @@
 name: ci
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: ['*']
 jobs:
+  lint:
+    runs-on: 'ubuntu-22.04'
+    steps:
+      - name: 'Get Changed Files'
+        id: 'files'
+        uses: 'jitterbit/get-changed-files@v1'
```

Comment:
I see some warnings in https://github.com/tj/git-extras/actions/runs/3979079395/jobs/6821262963
```
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

Can it be avoided?

Reply:
I switched to `masesgroup/retrieve-changed-files@v2`, which fixed this issue